README.md

```diff
@@ -1,2 +1,20 @@
 # Who we are?
 World Wide JUGs
+
+
+# Communication Channels
+Feel free to reach out using one of the following channels:
+  
+* Official mailing list: jug-leaders@jugs.groups.io
+* Slack: jugleaders.slack.com
+* https://github.com/wwjugs/Global
```

Comment:
```suggestion
* Github issues: https://github.com/wwjugs/Global
```

Reply:
Thanks @jonathanvila, I added this change in 0b5ce2e